stardis/radiation_field/opacities/base.py

```diff
@@ -0,0 +1,26 @@
+class Opacities:
+    """
+    Holds opacity information.
+
+    ###TODO: Change to dict that holds each of the sources of opacity separately. Then have a method that combines them all to return the total opacity.
+    Paramaters
+    ----------
+    opacities : dict
+        Python dictionary to contain each of the sources of opacity by name as well as their contribution at each frequency specified in the radiation field.
+    total_alphas : numpy.ndarray
+        Array of the total opacity at each frequency specified in the radiation field at each depth points.
+        Added as an attribute when calc_total_alphas() is called.
+    """
+
+    def __init__(self):
+        self.opacities = {}
```

Comment:
I wonder if this would be better called `opacities_dict` so the type is clear and avoids the `opacities.opacities` in other files.

Reply:
I like the idea a lot. I've changed the code accordingly.